tools/cellxgene_census_builder/src/cellxgene_census_builder/build_soma/summary_cell_counts.py

```diff
@@ -94,6 +95,7 @@ def accumulate_summary_counts(current: pd.DataFrame, obs_df: pd.DataFrame) -> pd
                 columns="is_primary_data",
                 index=["organism", "ontology_term_id", "label"],
                 fill_value=0,
+                aggfunc="sum",
```

Comment:
Curious why this wasn't specified before?

Reply:
the aggfunc is a noop no matter what it is specified as, because each value is unique (i.e., no aggregation occurs). However, the default aggfunc ('mean') causes the int values to be cast to a float. As the dataframe column is defined as int, this is no good. The sum aggfunc is specified because Python sum will leave ints as ints, bypassing the cast.

This was a latent bug I only found when the new TableSpec/FieldSpec class actually checked that the resulting schema was as expected.  Hence the fix.

Reply:
and I'll add a comment to this effect

Reply:
Thanks! This explains it.